buildSrc/src/main/java/com/osacky/flank/gradle/FladlePluginDelegate.kt

```diff
@@ -40,6 +40,8 @@ class FladlePluginDelegate {
       // Must be done afterEvaluate otherwise extension values will not be set.
       project.dependencies.add(FLADLE_CONFIG, "${base.flankCoordinates.get()}:${base.flankVersion.get()}")
 
+      if (project.hasProperty("sanityRobo")) base.sanityRobo.set(true)
```

Comment:
Reading project properties in this way is not compatible with the configuration cache. Can you please remove this or [make it compatible with the configuration cache](https://docs.gradle.org/current/userguide/configuration_cache.html#config_cache:requirements:undeclared_gradle_prop_read)?
The order of precedence for this project property also needs to be specified in the documentation.

Reply:
Fladle needs to support 6.2 as a minimum version https://docs.gradle.org/current/javadoc/org/gradle/api/provider/ProviderFactory.html#gradleProperty-java.lang.String-
For now, fladle will rely on extension property